the-open-university-s390.csl

```diff
@@ -0,0 +1,263 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>The Open University - S390</title>
```

Comment:
We could put back "Harvard" in the style title, but then we should update the other Open University course-specific Harvard variants as well (https://www.zotero.org/styles?q=open-university).

Reply:
Agree with the name change removing "Harvard" - 
(1) the "Harvard" part of the title is probably not necessary for a course-specific variant, 
(2) the variant refers to the original "Harvard"-based style brief (at line 8), and, 
(3) changing the other course styles to match these minor adjustments in presentation of the style seems overkill.
Finally, seeing as some of the other course specific variants have been around a few years, dropping "Harvard" from the title does not seem to have been an issue.